packages/lesswrong/components/recentDiscussion/RecentDiscussionSubscribeReminder.tsx

```diff
@@ -107,7 +125,7 @@ const RecentDiscussionSubscribeReminder = ({classes}: {
   >
     Maybe Later
   </Button>
-  
+
```

Comment:
I don't feel strongly about this so I'll change my settings but... we can just resolve the conflicts and have cleaner files without extra whitespace? :)

Reply:
1. That would be a pretty conflict-y PR that would make life worse for a while. (You could claim it'd be worth it, and I agree eventually it would, but it'd probably take years to pay off.)
2. More importantly you'd need to convince everyone working on LW to do it. Last time I tried to do that I lost.

If you did want to have a go at convincing LW, I'd be super excited to go all the way and use prettier or some other code formatter.

Reply:
Probably not the best way to introduce myself to them :)